modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerTest.java

```diff
@@ -401,15 +401,15 @@ public void testCreateTable() throws Exception {
     public void testWriteTableAssignmentsToMetastoreExceptionally() throws Exception {
         TableViewInternal table = mockManagersAndCreateTable(DYNAMIC_TABLE_NAME, tblManagerFut);
         int tableId = table.tableId();
-        TableManager tableManager = tblManagerFut.join();
+        TableAssignmentsManager assignmentsManager = new TableAssignmentsManager(msm, catalogManager, distributionZoneManager);
```

Comment:
Should we wait for that future anyway? Are you sure you haven't introduced some data race with this change?